docs/Parameters.rst

```diff
@@ -1124,7 +1124,9 @@ GPU Parameters
 
 -  ``gpu_use_dp`` :raw-html:`<a id="gpu_use_dp" title="Permalink to this parameter" href="#gpu_use_dp">&#x1F517;&#xFE0E;</a>`, default = ``false``, type = bool
 
-   -  set this to ``true`` to use double precision math on GPU (by default single precision is used in OpenCL implementation and double precision is used in CUDA implementation)
+   -  set this to ``true`` to use double precision math on GPU (by default single precision is used)
+
+   -  **Note**: can be used only in OpenCL implementation, in CUDA implementation only double precision is supported yet
```

Comment:
```suggestion
   -  **Note**: can be used only in OpenCL implementation, in CUDA implementation only double precision is currently supported
```

Reply:
Fixed in 4a175a9625fe2542abb600bcd6647927157b4dbe.